app/models/metric/rollup.rb

```diff
@@ -266,9 +266,14 @@ def self.rollup_realtime_perfs(obj, rt_perfs, new_perf = {})
     new_perf
   end
 
+  # @param obj parent object
+  # @param timestamp [Time]
+  # @param interval_name  ["realtime", "hourly", "historical"]
+  # @param assoc [Symbol] name of the association to rollup
   def self.rollup_child_metrics(obj, timestamp, interval_name, assoc)
-    ts = timestamp.kind_of?(Time) ? timestamp.utc.iso8601 : timestamp
-    recs = obj.vim_performance_state_association(timestamp, assoc)
+    timestamp = Time.parse(timestamp).utc if timestamp.kind_of?(String)
+    ts = timestamp.utc.iso8601
```

Comment:
We might want to rename this to `iso_ts` to be consistent with the code in `vim_performance_state_for_ts`.  Not a big deal either way though.